src/content_script/respondWithoutPrompt.ts

```diff
@@ -63,7 +63,7 @@ async function handleAutoPayment(msg: PaymentPromptMessage) {
   }
 
   // Don't allow payments to happen too fast
-  const last = lastPaymentAttempt;
+  const last = allowance.lastPaymentAttempt || lastPaymentAttempt;
```

Comment:
This should probably be `Math.max`, we probably want to defer to whichever is most recent.

Reply:
actually I think I can remove the `|| lastPaymentAttempt` because `allowance.lastPaymentAttempt` should be `0` for new allowances: https://github.com/joule-labs/joule-extension/pull/253/files#diff-34fdfa4008f4ad6d136ac8f4335e804aR259
and then we can rely on `allowance.lastPaymentAttempt` being updated.

Reply:
changed it to rely on the data persisted in the allowance config.